libfreerdp/core/license.c

```diff
@@ -1551,8 +1556,9 @@ BOOL license_send_platform_challenge_response_packet(rdpLicense* license)
  * @param license license module
  */
 
-BOOL license_send_valid_client_error_packet(rdpLicense* license)
+BOOL license_send_valid_client_error_packet(rdpRdp* rdp)
```

Comment:
Any specific reason to change the argument?

Reply:
Yep, we don't have access to the `rdpLicence` in the `rdpRdp`.

Reply:
`rdpLicense` is opaque there, correct. But you can still access all members of `rdpRdp` from where it is called.
Anyway, no blocker, just curious.